lib/translation_maps/library_code_translations.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+{
```

Comment:
This seems suspiciously like our `library_map.properties` file?

Reply:
Yep. Looks like there are two minor differences. Should we consolidate now?

Reply:
Though I wonder if the "lack of" the 13 additional keys at the end could cause problems.. I think that is why we just ported over what we had for now.

Reply:
Ah, I didn't scroll down to the bottom. I'll push the green button and let @jkeck chime in next week.